tests/pr_test.go

```diff
@@ -21,9 +23,11 @@ import (
 )
 
 // Ensure every example directory has a corresponding test
-const instanceFlavorDir = "solutions/deploy"
+const instanceFlavorDir = "solutions/fully-configurable"
 
-var permanentResources map[string]interface{}
+const cpdEntitlementKeyId = "a4292c24-f093-2b8b-9016-37132b7b8788"
```

Comment:
Initially this looked like a sensitive value, but its not. Maybe renamed and also add a code comment
cpdEntitlementKeyId -> cpdEntitlementKeySecretId